doc/source/whatsnew/v0.17.1.txt

```diff
@@ -202,3 +202,4 @@ Bug Fixes
 - Bug in ``DataFrame.to_sparse()`` loses column names for MultiIndexes (:issue:`11600`)
 - Bug in ``DataFrame.round()`` with non-unique column index producing a Fatal Python error (:issue:`11611`)
 - Bug in ``DataFrame.round()`` with ``decimals`` being a non-unique indexed Series producing extra columns (:issue:`11618`)
+- Fixed PY2/PY3 incompatability in ``Series.items`` (:issue:`13918`) 
```

Comment:
put this in the API changes section as that will gain more attention. Say ``Series`` has gained the ``.items()`` method to mirror ``.iteritems()`` and for compat with PY3 (and add for ``DataFrame``) as well.